manifests/charts/gateway/values.yaml

```diff
@@ -95,3 +95,6 @@ networkGateway: ""
 imagePullPolicy: ""
 
 imagePullSecrets: []
+
+defaultPodDisruptionBudget:
```

Comment:
Why is it **default**PodDisruptionBudget? Either there is a PDB or there isn't, not sure what "default" means here?

Reply:
I copied this from the `gateways` chart, and it appears that it was originally "podDisruptionBudget" but eventually changed to "defaultDisruptionBudget." Regardless, I have revised it to PDB for better clarity. Thanks.